utils/modify_label_json_file.py

```diff
@@ -51,16 +51,20 @@ def substitute_matching_char():
         json.dump(l_label, fp, indent=4)
 
 
-def custom_change_field(json_file):
-    with Path(json_file).open(mode='r', encoding='utf-8') as fp:
+def custom_change_field():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("file", type=str, help="Path to the json file to be modified")
+    args = parser.parse_args()
+    with Path(args.file).open(mode='r', encoding='utf-8') as fp:
         d_label = json.load(fp)
     for img_id, label in d_label.items():
-        label["label_fingerprint"] = utils_fct.get_label_finger_print(label)
-    with Path(json_file).open(mode='w', encoding='utf-8') as fp:
+        # label["label_fingerprint"] = utils_fct.get_label_finger_print(label)
```

Comment:
do you want to keep the commented line ?

Reply:
cette fonction est a utiliser uniquement en modifiant le code lui faire faire ce qu'on veut, c'est du bidouillage, on pourra la rendre plus 'propre' si besoin